specification/network/resource-manager/Microsoft.Network/stable/2020-06-01/firewallPolicy.json

```diff
@@ -838,7 +847,14 @@
           "items": {
             "type": "string"
           }
-        }
+        },
+        "destinationWebCategories": {
```

Comment:
We don't allow changes to existing api-versions in the master branch.  Did you intend to add this to a new api-version?

Reply:
Also, you will need to resolve the prettier issue below by running prettier locally

Reply:
![image](https://user-images.githubusercontent.com/4796246/95129186-7ee87700-070f-11eb-9925-c55b92bff461.png)
Yes. this is completed